public/gameInfo.php

```diff
@@ -560,36 +569,57 @@ function submitSetRequest(user, gameID) {
             $imageIngame = $gameData['ImageIngame'];
             $pageTitleAttr = attributeEscape($pageTitle);
 
+            function addMetadata($label, $gameDataValue, $gameHubs)
```

Comment:
As this function renders HTML I'd probably put it in the render lib folder - even though this application is only used here atm. Yet, I don't think we keep functions within "endpoint" files anywhere else.
I'd suggest `renderMetadata()` for the function name as it does in fact render, not add metadata?
That'd be mostly in line with the rest of the application, too.

Reply:
The function was created to reuse code. It's kind of awkward to put that in a different file as it's only generating a single table row, but I did it anyway.